tests/filesystem/AssetStoreTest.php

```diff
@@ -1,10 +1,12 @@
 <?php
```

Comment:
This whole test could use something like [vfsStream](Filesystem) to don't write/read directly from the filesystem (which is not a good practice in software testing)